### ISSUE TITLE
Missing icon for Cluster2Node component 

### ISSUE THREAD

**Issue by al-fisher** [BHoM/BHoM_UI#232]
Need to create and add new icon for **Cluster2Node**

@adecler - now's a good time to review our general icon assets actually to just tie into the rest of the BHoM logos, iconography, website material etc. 

perhaps you and I grab a quick session to coordinate? 😄

**Comment by adecler** [comment]
Yes, we also need a proper icon for BHoM Adapter parameter component.
